src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -188,6 +219,21 @@ export default class ChargingStationService {
         'chargingStationURL': chargingStation.chargingStationURL
       }
     });
+    if (rebootRequired) {
```

Comment:
Reboot only when flag manualConfiguration is set to false.

Reply:
I still do not really understand why a reboot is required here. 
The only reason to reboot the charging station is if a change is made to an OCPP parameter that returns reboot required message.